### ISSUE TITLE
Will you release the model training code?

### ISSUE THREAD

**Issue by JustinhoCHN** [xinntao/ESRGAN#2]
I want to see how you guys implement the loss function. It'll be great if you release the training code. Thank you.
